integration_tests/dynamodb/create_sink.sql

```diff
@@ -0,0 +1,13 @@
+CREATE SINK dyn_sink
+FROM
+  movies
+WITH
+(
+  connector = 'dynamodb',
+  table = 'Movies',
+  primary_key = 'year,title',
```

Comment:
Is this field redundant? I didn't find this config in the generated sink configuration yaml.

Reply:
The config validator will validate the fields `primary_key` to ensure that they are primary keys in rw's sink_from table and dynamodb table. This is indeed not required for the sink itself to work, but currently the config validator requires this field to be required.

Reply:
I see. Added to the doc.